document/i18n/ja/docusaurus-plugin-content-docs/current/bulkmessage.md

```
@@ -33,4 +33,4 @@ LINE連携カラムに表示されるLINE表示名がリンクとなっており
 対象ユーザーのチェックボックスにチェックを入れ、一括操作で「LINE送信する」を選択し「適用」ボタンをクリックします。  
 <img src="https://blog.shipweb.jp/wp-content/uploads/2022/01/lineconnect-ss-12.jpg" width="320" />  
 チェックしたWordPressユーザーIDが指定された状態で一括配信画面が開きます。  
-<img src="https://blog.shipweb.jp/wp-content/uploads/2022/01/lineconnect-ss-13.jpg" width="320" />  
\ No newline at end of file
+<img src="https://blog.shipweb.jp/wp-content/uploads/2022/01/lineconnect-ss-13.jpg" width="320" />  
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)